test/integration/transport_scene_manager.cc

```diff
@@ -20,6 +20,7 @@
 #include <gz/common/Console.hh>
 #include <gz/math/Color.hh>
 #include <gz/math/Pose3.hh>
+#include <gz/msgs/uint32_v.pb.h>
```

Comment:
is this required?

Reply:
No, it's part of other work, will remove for now.

Reply:
it's still there